.github/workflows/ci.yml

```diff
@@ -346,6 +346,23 @@ jobs:
       - name: Upload test coverage to Codecov
         uses: codecov/codecov-action@v1.0.12
 
+  spark_integration_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0  # fetch all tags and branches
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Install test dependencies
+        run: ./ci/install_test_deps.sh
```

Comment:
could you update this line  to `./ci/install_test_deps.cmd` to reflect changes in https://github.com/bentoml/BentoML/pull/1546